core/src/scene/drawRule.cpp

```diff
@@ -21,6 +21,16 @@ const std::map<std::string, StyleParamKey> s_StyleParamMap = {
     {"outline:width", StyleParamKey::outline_width},
     {"outline:cap", StyleParamKey::outline_cap},
     {"outline:join", StyleParamKey::outline_join},
+    {"font:name", StyleParamKey::font_name},
+    {"font:weight", StyleParamKey::font_weight},
+    {"font:face", StyleParamKey::font_face},
+    {"font:size", StyleParamKey::font_size},
+    {"font:fill", StyleParamKey::font_fill},
+    {"font:stroke", StyleParamKey::font_stroke},
+    {"font:stroke_color", StyleParamKey::font_stroke_color},
+    {"font:stroke_width", StyleParamKey::font_stroke_width},
+    {"font:capitalized", StyleParamKey::font_capitalized},
+    {"font:visible", StyleParamKey::font_visible}
```

Comment:
Separate toggle for label visibility in general and text visibility as a "sub layer" seems useful to me. I've setup Skin and Bones this way thru some hacks.

Sent from my handsful device.



> On Aug 28, 2015, at 08:26, Brett Camper <notifications@github.com> wrote:
> 
> In core/src/scene/drawRule.cpp:
> 
> > @@ -21,6 +21,16 @@ const std::map<std::string, StyleParamKey> s_StyleParamMap = {
> >      {"outline:width", StyleParamKey::outline_width},
> >      {"outline:cap", StyleParamKey::outline_cap},
> >      {"outline:join", StyleParamKey::outline_join},
> > +    {"font:name", StyleParamKey::font_name},
> > +    {"font:weight", StyleParamKey::font_weight},
> > +    {"font:face", StyleParamKey::font_face},
> > +    {"font:size", StyleParamKey::font_size},
> > +    {"font:fill", StyleParamKey::font_fill},
> > +    {"font:stroke", StyleParamKey::font_stroke},
> > +    {"font:stroke_color", StyleParamKey::font_stroke_color},
> > +    {"font:stroke_width", StyleParamKey::font_stroke_width},
> > +    {"font:capitalized", StyleParamKey::font_capitalized},
> > +    {"font:visible", StyleParamKey::font_visible}
> Not sure we need a separate visible flag for font blocks specifically. What's the thinking here? (Maybe it's a debug thing?)
> 
> ―
> Reply to this email directly or view it on GitHub.
>